testing/compile/fork_join_cdp_deprecation/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.15)
+
+# Remove this when we use the new CUDA_ARCHITECTURES properties with both
+# nvcc and nvc++.
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.18)
+  cmake_policy(SET CMP0104 OLD)
+endif()
+
+project(fork_join_cdp_deprecation CXX CUDA)
+
+message(STATUS "Thrust_DIR=${Thrust_DIR}")
+
+# Need C++14 for deprecation messages
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CUDA_STANDARD 14)
+
+find_package(Thrust)
```

Comment:
I would make `find_package(Thrust)` Required so that you get a nice error message on the find call instead on the  `thrust_create_target`